extras/format.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 # clang formatting
 echo "Start formatting source files.."
+# find . -not -path "./.*" -name "*.cpp" -o -name "*.c" -o -name "*.h" | xargs -I {} clang-format -i {}
+# find . -type d \( -path "./.*" -o -path "./datasheets" -o -path "./medis" \) -prune -false -o -name "*.h" -o -name "*.hpp" -o -name "*.cpp" | xargs clang-format -i {}
```

Comment:
I assume this is a leftover?

Reply:
The original command doens't work on my machine. The one on line 5 does though.

Reply:
Ok strange. What version are you using (clang-format) and what is the error message?

Reply:
10.0.0-4ubuntu1